### DISCUSSION TITLE
Possibility to have all values of listed entities not only that is toggeled

### DISCUSSION THREAD

**Discussion by ellvinib** [ExperienceLovelace/ha-floorplan#423] [Ideas]
Hi, it would be nice to have the list of all entities in the service_data
that i can do "entities.entities_ids" and i will get a list of all "entities"

`- type: custom:floorplan-card
                full_height: true
                config:
                  image: /local/floorplan/floor-0.svg?v=2
                  stylesheet: /local/floorplan/floorplan.css
                  defaults:
                    hover_action: hover-info
                    tap_action: more-info
                  rules:
                    - entities:
                        - light.lamp_leefruimte_keuken_werktafel
                        - light.lamp_inkom_1
                        - light.lamp_leefruimte_keuken_eettafel
                        - light.lamp_douche
                        - switch.met_binnenmelder_s_1
                        - switch.met_binnenmelder_s_2
                        - switch.met_binnenmelder_s_4
                        - light.lamp_leefruimte_zithoek
                        - light.lamp_voorgevel
                        - light.lamp_leefruimte_sfeer
                        - light.lamp_leefruimte_grondspots
                        - light.lamp_buiten_zijgevel_links
                      state_action:
                        - action: call-service
                          service: floorplan.style_set
                          service_data: |
                            >

                            here i want to see wich lamps in the entities are 'on' so i can deduce wich overlay i should choose ( i have 10 lights in my room and i have 72 combination overlays)
                            i do not want to re define them here i want to access them like you can access the toggeled value entity.entity_id
                            entities.entities_id => all the ids of above ?
                            
                              
                              return {
                                elements: elements,
                                style: style,
                              };
                      tap_action:
                        action: toggle`

**Comment by ellvinib** [comment]
Nevermind my idea, i can achieve it trough config variables

**Reply by exetico** [reply]
Consider sharing your solution to other people, by sharing it here, in the same thread.

But we have entities exposed as a global variable. That's mentioned in the docs too, and you're able to use the global config value, to calculate (grap) all related values to the triggered rule.
